qa/TL1_git_submodule/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+test_body() {
+    mkdir -p /tmp/repo_that_uses_dali
+    pushd /tmp/repo_that_uses_dali
+    git init
+    git submodule add https://github.com/NVIDIA/DALI
+    pushd DALI
+    git submodule sync --recursive
+    git submodule update --init --recursive
+    popd
+    touch main_stub.cc
+    echo "
```

Comment:
Again I would prefer file instead of echo.

Reply:
Done